src/App.tsx

```diff
@@ -8,7 +8,58 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+
+type TodoSelectHandler = (todoId: number) => void;
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [dataIsLoaded, setDataIsLoaded] = useState(false);
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [todoStatus, setTodoStatus] = useState<TodoStatus>(TodoStatus.All);
+  const [query, setQuery] = useState('');
+
+  const loadTodos = async () => {
+    const todosFromServer = await getTodos();
+
+    setTodos(todosFromServer);
+    setDataIsLoaded(true);
+  };
```

Comment:
Well, better to use useCallback here, I think we don't want to create this function on every render